server/services/licence/licenceStatus.ts

```diff
@@ -372,38 +353,16 @@ function getLicenceConditionsState(licence) {
   }
 }
 
-function getSeriousOffenceCheckState(licence) {
-  const seriousOffenceAnswer = getIn(licence, ['finalChecks', 'seriousOffence', 'decision'])
-
-  return {
-    seriousOffence: seriousOffenceAnswer && seriousOffenceAnswer === 'Yes',
-    seriousOffenceCheck: seriousOffenceAnswer ? taskState.DONE : taskState.UNSTARTED,
-  }
-}
-
-function getOnRemandCheckState(licence) {
-  const onRemandAnswer = getIn(licence, ['finalChecks', 'onRemand', 'decision'])
-
-  return {
-    onRemand: onRemandAnswer && onRemandAnswer === 'Yes',
-    onRemandCheck: onRemandAnswer ? taskState.DONE : taskState.UNSTARTED,
-  }
-}
-
-function getConfiscationOrderState(licence) {
-  const confiscationOrderAnswer = getIn(licence, ['finalChecks', 'confiscationOrder', 'decision'])
-
-  return {
-    confiscationOrder: confiscationOrderAnswer && confiscationOrderAnswer === 'Yes',
-    confiscationOrderCheck: confiscationOrderAnswer ? taskState.DONE : taskState.UNSTARTED,
-  }
-}
+const getTaskState = (answer: any) => ({
+  decision: answer === 'Yes',
+  task: answer ? taskState.DONE : taskState.UNSTARTED,
+})
```

Comment:
extracted out this more generic method to replace 4 different `get*State` methods